<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dashboard</class>
 <widget class="QWidget" name="Dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1950</width>
    <height>1500</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*background-color: rgb(143, 180, 207);*/
/*background-color: rgb(30, 144, 255);*/
/*background-color: rgb(26, 26, 42);*/
/*background-color: rgb(74, 74, 74);*/
/*background-color: rgb(138, 138, 255);*/
background-color: rgb(114, 159, 207);


</string>
  </property>
  <widget class="QPushButton" name="newUserButton">
   <property name="geometry">
    <rect>
     <x>705</x>
     <y>930</y>
     <width>160</width>
     <height>47</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/*background-color: rgb(255, 132, 77);*/
QPushButton {
    background-color: rgb(255, 132, 77); /* Orange background */
    color: rgb(0, 0, 0); /* Black text */
    border: 2px solid rgb(205, 92, 92); /* Indian Red border */
    padding: 10px 20px; /* Padding for better appearance */
    border-radius: 5px; /* Rounded corners for a modern look */
    font-size: 16px; /* Adjust font size */
    font-weight: bold; /* Bold text */
    transition: background-color 0.3s ease, border-color 0.3s ease; /* Smooth transition */
}

QPushButton:hover {
    background-color: rgb(255, 69, 0); /* Red-Orange on hover */
}

QPushButton:focus {
    border-color: rgb(255, 255, 255); /* White border on focus */
    box-shadow: 0 0 5px rgba(255, 255, 255, 0.5); /* Optional shadow on focus */
}
</string>
   </property>
   <property name="text">
    <string>Add New User</string>
   </property>
  </widget>
  <widget class="QPushButton" name="adminUserButton">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>930</y>
     <width>180</width>
     <height>47</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/*background-color: rgb(255, 132, 77);*/
QPushButton {
    background-color: rgb(255, 132, 77); /* Orange background */
    color: rgb(0, 0, 0); /* Black text */
    border: 2px solid rgb(205, 92, 92); /* Indian Red border */
    padding: 10px 20px; /* Padding for better appearance */
    border-radius: 5px; /* Rounded corners for a modern look */
    font-size: 16px; /* Adjust font size */
    font-weight: bold; /* Bold text */
    transition: background-color 0.3s ease, border-color 0.3s ease; /* Smooth transition */
}

QPushButton:hover {
    background-color: rgb(255, 69, 0); /* Red-Orange on hover */
}

QPushButton:focus {
    border-color: rgb(255, 255, 255); /* White border on focus */
    box-shadow: 0 0 5px rgba(255, 255, 255, 0.5); /* Optional shadow on focus */
}
</string>
   </property>
   <property name="text">
    <string>Add New Admin </string>
   </property>
  </widget>
  <widget class="QTableWidget" name="dashboardTableWidget">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>60</y>
     <width>1891</width>
     <height>851</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
    background-color: rgb(26,26,42); /* Cornflower Blue background */
    color: rgb(255,255,255); /* Black text */
    gridline-color: rgb(255,255,255); /* Black grid lines */
    font-family: Arial, sans-serif; /* Modern, clean font */
}
QHeaderView::section {
    background-color: rgb(65, 105, 225); /* Royal Blue header background */
    color: rgb(255,255,255); /* White text for headers */
    font-weight: bold; /* Bold text for headers */
    border: 1px solid rgb(255, 255, 255); /* White border for headers */
}
QTableWidget::item:alternate {
    background-color: rgb(173, 216, 230); /* Light Blue for alternate rows */
}

QTableWidget::item:selected {
    background-color: rgb(255, 215, 0); /* Gold for selected items */
    color: rgb(0, 0, 0); /* Black text for selected items */
}

QTableWidget::item:hover {
    background-color: rgb(135, 206, 250); /* Light Sky Blue for hovered items */
}
</string>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="columnCount">
    <number>6</number>
   </property>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>39</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>315</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>26</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>33</number>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>User Id</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="background">
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>First Name</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Last Name</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Security Question</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Answer</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Role</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>351</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="CustomLineEdit" name="searchLineEdit">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">/*background-color: rgb(158, 109, 97,0%);*/
 background-color: rgba(255, 255, 255, 0.8);
color: rgb(0, 0, 0);
border: 1px solid rgb(100, 100, 100);
 padding: 5px;
 border-radius: 5px;</string>
      </property>
      <property name="placeholderText">
       <string>Enter text to search from table</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="searchButton">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">/*background-color: rgb(255, 132, 77);*/
QPushButton {
    background-color: rgb(255, 132, 77); /* Orange background */
    color: rgb(0, 0, 0); /* Black text */
    border: 2px solid rgb(205, 92, 92); /* Indian Red border */
    padding: 10px 20px; /* Padding for better appearance */
    border-radius: 5px; /* Rounded corners for a modern look */
    font-size: 16px; /* Adjust font size */
    font-weight: bold; /* Bold text */
    transition: background-color 0.3s ease, border-color 0.3s ease; /* Smooth transition */
}

QPushButton:hover {
    background-color: rgb(255, 69, 0); /* Red-Orange on hover */
}

QPushButton:focus {
    border-color: rgb(255, 255, 255); /* White border on focus */
    box-shadow: 0 0 5px rgba(255, 255, 255, 0.5); /* Optional shadow on focus */
}
</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="saveToPdfButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*background-color: rgb(255, 132, 77);*/
QPushButton {
    background-color: rgb(255, 132, 77); /* Orange background */
    color: rgb(0, 0, 0); /* Black text */
    border: 2px solid rgb(205, 92, 92); /* Indian Red border */
    padding: 10px 20px; /* Padding for better appearance */
    border-radius: 5px; /* Rounded corners for a modern look */
    font-size: 16px; /* Adjust font size */
    font-weight: bold; /* Bold text */
    transition: background-color 0.3s ease, border-color 0.3s ease; /* Smooth transition */
}

QPushButton:hover {
    background-color: rgb(255, 69, 0); /* Red-Orange on hover */
}

QPushButton:focus {
    border-color: rgb(255, 255, 255); /* White border on focus */
    box-shadow: 0 0 5px rgba(255, 255, 255, 0.5); /* Optional shadow on focus */
}
</string>
   </property>
   <property name="text">
    <string>Save to Pdf</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">customlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>searchLineEdit</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>dashboardTableWidget</tabstop>
  <tabstop>newUserButton</tabstop>
  <tabstop>adminUserButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
